File: projects/order-or-localserver/src/main/java/com/robinfood/orderorlocalserver/entities/orderdetail/OrderDetailPaymentMethodEntity.java
package com.robinfood.orderorlocalserver.entities.orderdetail;

import lombok.Data;

@Data
public class OrderDetailPaymentMethodEntity {

    private final Double discount;

    private final Long id;

    private final Long originId;

    private final Double subtotal;

    private final Double tax;

    private final Double value;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/OrderDetailPaymentMethodEntity.kt
package com.robinfood.core.entities

import java.math.BigDecimal

data class OrderDetailPaymentMethodEntity(
        val discount: BigDecimal?,
        val id: Long?,
        val originId: Long?,
        val subtotal: BigDecimal?,
        val tax: BigDecimal?,
        val value: BigDecimal?
)
